### DISCUSSION TITLE
Issue while Upgrading Serverless with @3.0.1 and @3.0.0

### DISCUSSION THREAD

**Discussion by SaitejaChavva** [serverless/serverless#10613] [Q&A]
Hi Team,

I was facing issue while upgrading Serverless version with @3.0.1 and @3.0.0. 
PFB snapshot's of error message
**serverless@3.0.1**
![image](https://user-images.githubusercontent.com/67137264/151943227-4753c322-d8d0-4ad2-a0e2-7e8097684ae0.png)

In jenkins-deploy phase, we are facing below issue while upgrading the serverless version

19:42:13  Environment: linux, node 12.14.1, framework 3.0.1 (local), plugin 6.0.0, SDK 4.3.0
19:42:13  Credentials: Local, environment variables
19:42:13  Docs:        docs.serverless.com
19:42:13  Support:     forum.serverless.com
19:42:13  Bugs:        github.com/serverless/serverless/issues
19:42:13  
19:42:13  Error:
19:42:13  When external API Gateway resource is imported via "provider.apiGateway.restApiId", property "provider.tracing.apiGateway" is ineffective.
19:42:13  
19:42:13  3 deprecations found: run 'serverless doctor' for more details
19:42:13  make: *** [Makefile.deployServerless:25: deploy] Error 1
[Pipeline] }
[Pipeline] // withFolderProperties
[Pipeline] }
[Pipeline] // stage
[Pipeline] }
[Pipeline] // maskPasswords
[Pipeline] createSummary
[Pipeline] sh (hid

**serverless@3.0.0**
![image](https://user-images.githubusercontent.com/67137264/151942975-645bb12f-f1de-43fe-bb92-163599d71946.png)

13:36:30  Environment: linux, node 12.14.1, framework 3.0.0 (local), plugin 6.0.0, SDK 4.3.0
13:36:30  Credentials: Local, environment variables
13:36:30  Docs:        docs.serverless.com
13:36:30  Support:     forum.serverless.com
13:36:30  Bugs:        github.com/serverless/serverless/issues
13:36:30  
13:36:30  Error:
13:36:30  When external API Gateway resource is imported via "provider.apiGateway.restApiId", property "provider.tracing.apiGateway" is ineffective.
13:36:30  
13:36:30  3 deprecations found: run 'serverless doctor' for more details
13:36:31  make: *** [Makefile.deployServerless:25: deploy] Error 1
[Pipeline] }
[Pipeline] // withFolderProperties
[Pipeline] }
[Pipeline] // stage
[Pipeline] }
[Pipeline] // maskPasswords
[Pipeline] createSummary

Please give me any suggestions and do the needful, to fix the issue

**Comment by ahmednuaman** [comment]
I "fixed" it by simply removing the option from `serverless.yml`

```diff
   architecture: arm64
   stage: ${opt:stage, 'development'}
   region: ${env:AWS_DEFAULT_REGION, 'eu-west-1'}
-  tracing:
-    apiGateway: true
   timeout: 60
   memorySize: 512
```

**Comment by thalesfsp** [comment]
NOTE for future people looking for `When external API Gateway resource is imported via "provider.apiGateway.restApiId", property "provider.tracing.apiGateway" is ineffective`. **This is the way to solve.**
